packages/components/src/Avatar/Avatar.stories.mdx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Tooltip } from "@jobber/components/Tooltip";
+import { Avatar } from "@jobber/components/Avatar";
+
+<Meta title="Components/Avatar" component={Avatar} />
+
+# Avatar
+
+An avatar will be used to display an visual identifier for an individual user.
+
+```ts
+import { Avatar } from "@jobber/components/Avatar";
+```
+
+<Canvas>
+  <Story name="Avatar">
+    {() => {
+      return (
+        <Avatar
+          imageUrl="https://images.unsplash.com/photo-1533858539156-90ea20bafd17?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80"
+          name="The Jobbler"
+          size="large"
+          initials="JBLR"
+          color="var(--color-indigo)"
+        />
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Design & usage guidelines
+
+- Avatars are available in 3 sizes:
+  - `base`: Used by default.
+  - `large`: Use when the avatar is the focal point.
+  - `small`: Use on higher-density/compact pages or components.
+- If an avatar is displayed without a name label, display a tooltip.
+- Initials and background color should have sufficient color contrast to meet
+  AA.
+
+## Props
+
+<Props of={Avatar} />
+
+## Color
+
+The concept of `color` with an Avatar is currently not used in Jobber.
+
+Use `color` to indicate the background and the border color of an avatar
+component.
+
+When using `color`, make sure that the color is consistent for a user throughout
+the application. The color that is represented for an avatar should be the same
+color that represents a user in other parts of the application, such as the
+calendar.
+
+<Canvas>
+  <Story name="Color">
+    {() => {
```

Comment:
don't need to put everything under a story 

https://github.com/GetJobber/atlantis/blob/640ebd2d5e126c1a0dc5ab323bfbbab3a000676e/packages/components/src/Button/Button.stories.mdx?plain=1#L57-L59